packages/auth/README.md

```diff
@@ -5,7 +5,7 @@ Functions to help authenticate with the Coveo platform.
 ## Install
 
 ```
-npm i @coveo/saml
+npm i @coveo/auth
 ```
 
 ## SAML
```

Comment:
Hey @dlutzCoveo, I would appreciate if you could look over this `README.md`.

Also, should we update [this page](https://docs.coveo.com/en/91/build-a-search-ui/saml-authentication) to mention this package? The first paragraph mentions the `AuthenticationProvider` component in the JSUI. This package would be relevant to non-JSUI users.

Reply:
Hi @samisayegh! README is 👍. 
I'll make a Jira to mention this package in the SAML doc page you linked above.